provider/workspace_preset.go

```diff
@@ -10,8 +10,13 @@ import (
 )
 
 type WorkspacePreset struct {
-	Name       string            `mapstructure:"name"`
-	Parameters map[string]string `mapstructure:"parameters"`
+	Name       string              `mapstructure:"name"`
+	Parameters map[string]string   `mapstructure:"parameters"`
+	Prebuild   []WorkspacePrebuild `mapstructure:"prebuilds"`
```

Comment:
Why we use `[]WorkspacePrebuild` instead of `WorkspacePrebuild`?

Reply:
I couldn't find a reason for this. I've changed it to just WorkspacePrebuild.